agent/command_test.go

```diff
@@ -82,6 +82,7 @@ func (s *CommandSuite) SetupTest() {
 		Identifier: "project_identifier",
 	}, &patch.Patch{}, util.Expansions{})
 	s.Require().NoError(err)
+	s.Equal(&util.Expansions{}, taskConfig.DynamicExpansions)
```

Comment:
This isn't wrong, but it's a little weird to drive-by unit test the behavior of `internal.NewTaskConfig` in the agent suite. It seems like if we want to test the behavior of `internal.NewTaskConfig` itself, we should explicitly test that in the `internal` package, rather than drive-by test it in the setup here.